packages/core/src/components/slider/slider.tsx

```diff
@@ -121,33 +132,33 @@ export class TdsSlider {
   }
 
   @Listen('mouseup', { target: 'window' })
-  handleMouseUp() {
+  handleMouseUp(event: MouseEvent) {
     if (!this.thumbGrabbed) {
       return;
     }
 
     this.thumbGrabbed = false;
     this.thumbInnerElement.classList.remove('pressed');
-    this.updateValue();
+    this.updateValue(event);
 
     this.trackElement.focus();
   }
 
   @Listen('touchend', { target: 'window' })
-  handleTouchEnd() {
+  handleTouchEnd(event: TouchEvent) {
```

Comment:
Not very critical, but Sonarcloud reports duplication on these two methods. Maybe something to refactor in another PR?
https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&pullRequest=513&id=scania-digital-design-system_tegel&open=AY3FsqZABUMWrAS_OREb

Maybe into a one method to hold the shared logic and the other ones can just call that one?

```
@Listen('mouseup', { target: 'window' })
handleMouseUp() {
  this.handleRelease();
}

@Listen('touchend', { target: 'window' })
handleTouchEnd() {
  this.handleRelease();
}

// New method that contains the shared logic
private handleRelease() {
  if (!this.thumbGrabbed) {
    return;
  }

  this.thumbGrabbed = false;
  this.thumbInnerElement.classList.remove('pressed');
  this.updateValue();
  this.trackElement.focus();
}
```

Reply:
Good call, I will check if I can sort it out in this one. Should not be biggy...

Reply:
Change pushed ;)